CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ Mapbox welcomes participation and contributions from everyone.
 - Moved `MapboxAudioGuidance` and `MapboxAudioGuidanceState` into public api. [#6336](https://github.com/mapbox/mapbox-navigation-android/pull/6336)
 - Introduced `ViewOptionsCustomization.showCameraDebugInfo` to allow end users to enable camera debug info. [#6356](https://github.com/mapbox/mapbox-navigation-android/pull/6356)
 - Added `ComponentInstaller` for the `LocationComponent` that offers simplified integration of map LocationPuck. [#6206](https://github.com/mapbox/mapbox-navigation-android/pull/6206)
+- Added _Experimental_ `RouteRefreshStatesObserver` that can be used to observe route refresh states. To subscribe and unsubscribe on updates corresponding use `MapboxNavigation#setRouteRefreshStateObserver` and `MapboxNavigation#resetRouteRefreshStateObserver`. [#6345](https://github.com/mapbox/mapbox-navigation-android/pull/6345)
```

Comment:
This can be removed.

Reply:
why? It's publci API that any of customers might start to use 🤔

Reply:
We have duplicated changelog entries now. This mention the removed `setRouteRefreshStateObserver`, we now have `registerRouteRefreshStateObserver` instead.

Reply:
nice catch. Rebase issue I think